src/logged-evaluation.nix

```diff
@@ -129,9 +129,14 @@ let
 
             # Export IN_NIX_SHELL to trick various Nix tooling to export
             # shell-friendly variables
-
             export IN_NIX_SHELL=impure
 
+            ${if shellSrc == null then "" else ''
+              # Export IN_LORRI_SHELL to differentiate between `lorri shell` and
+              # `nix-shell`
+              export IN_LORRI_SHELL=${shellSrc}
+            ''}
```

Comment:
The `null` conditional looks strange to me; @curiousleo do we still require this switch for the services? I think we only ever had a WIP for services, do we want to keep them around in the codebase?

Reply:
If `shellSrc ? null` turns out to be unnecessary (pending @curiousleo's response), should I just remove `? null` from the argset?

Reply:
I suggest we merge #405 first and rebase this PR on top of it. I've filed #404 for follow-up work to remove complexity due to the incomplete `lorri services` implementation.